src/Domain/CoordinateMaps/TimeDependent/SphericalCompression.hpp

```diff
@@ -0,0 +1,322 @@
+ * the unmapped coordinates with respect to the center \f$C^i\f$, \f$\rho =
+ * \sqrt{\delta_{kl}\left(\xi^k - C^l\right)\left(\xi^l - C^l\right)}\f$ is the
+ * Euclidean magnitude of \f$\rho^i\f$, and \f$\rho_j = \delta_{ij} \rho^i\f$.
+ *
+ * ### Frame velocity
+ *
+ * The frame velocity \f$v^i \equiv dx^i/dt\f$ is then
+ * \f{align}{
+ * v^i &= \left\{\begin{array}{ll} - \frac{\lambda_{00}^\prime(t)}{\sqrt{4\pi}}
+ * \frac{\rho^i}{\rho_{\rm min}}, & \rho < \rho_{\rm min}, \\
+ * - \frac{\lambda_{00}^\prime(t)}{\sqrt{4\pi}}
+ * \frac{\rho_{\rm max} / \rho - 1}{\rho_{\rm max} - \rho_{\rm min}} \rho^i,
+ * & \rho_{\rm min} \leq \rho \leq \rho_{\rm max}, \\
+ * 0, & \rho_{\rm max} < \rho.\end{array}\right.
+ * \f}
+ *
+ * ### Jacobian
+ *
+ * Differentiating the equations for \f$x^i\f$ gives the Jacobian
+ * \f$\partial x^i / \partial \xi^j\f$. Using the result
+ * \f{align}{
+ * \frac{\partial \rho^i}{\partial \xi^j} &= \frac{\partial}{\partial \xi^j}
+ * \left(\xi^i - C^i\right) = \frac{\partial \xi^i}{\partial \xi^j}
+ * = \delta^i_{j}
+ * \f}
+ * and taking the derivatives yields
+ * \f{align}{
+ * \frac{\partial x^i}{\partial \xi^j} &= \left\{\begin{array}{ll}
+ * \delta^i_j \left(1
+ * - \frac{\lambda_{00}(t)}{\sqrt{4\pi}} \frac{1}{\rho_{\rm min}}\right),
+ * & \rho < \rho_{\rm min},\\
+ * \delta^i_j
+ * \left(1 - \frac{\lambda_{00}(t)}{\sqrt{4\pi}}
+ * \frac{\rho_{\rm max} / \rho - 1}{\rho_{\rm max} - \rho_{\rm min}}\right)
+ * - \rho^i \frac{\lambda_{00}(t)}{\sqrt{4\pi}}
+ * \frac{\partial}{\partial \xi^j}\left(
+ * \frac{\rho_{\rm max} / \rho - 1}{\rho_{\rm max} - \rho_{\rm min}}\right),
+ * & \rho_{\rm min} \leq \rho < \rho_{\rm max},\\
+ * \delta^i_j, & \rho_{\rm max} < \rho.\end{array}\right.
+ * \f}
+ * Inserting
+ * \f{align}{
+ * \frac{\partial}{\partial \xi^j}\left(
+ * \frac{\rho_{\rm max} / \rho - 1}{\rho_{\rm max} - \rho_{\rm min}}\right)
+ * &= \frac{\rho_{\rm max}}{\rho_{\rm max} - \rho_{\rm min}}
+ * \frac{\partial}{\partial \xi^j}\left(\frac{1}{\rho}\right)
+ * = - \frac{\rho_{\rm max}}{\rho_{\rm max} - \rho_{\rm min}} \frac{1}{\rho^2}
+ * \frac{\partial \rho}{\partial \xi^j}
+ * \f}
+ * and
+ * \f{align}{
+ * \frac{\partial \rho}{\partial \xi^j} &= \frac{\rho_j}{\rho}.
+ * \f}
+ * into the Jacobian yields
+ * \f{align}{
+ * \frac{\partial x^i}{\partial \xi^j} &= \left\{\begin{array}{ll}
+ * \delta^i_j \left(1
+ * - \frac{\lambda_{00}(t)}{\sqrt{4\pi}} \frac{1}{\rho_{\rm min}}\right),
+ * & \rho < \rho_{\rm min},\\
+ * \delta^i_j
+ * \left(1 - \frac{\lambda_{00}(t)}{\sqrt{4\pi}}
+ * \frac{\rho_{\rm max} / \rho - 1}{\rho_{\rm max} - \rho_{\rm min}}\right)
+ * + \rho^i \rho_j \frac{\lambda_{00}(t)}{\sqrt{4\pi}}
+ * \frac{\rho_{\rm max}}{\rho_{\rm max} - \rho_{\rm min}}\frac{1}{\rho^3},
+ * & \rho_{\rm min} \leq \rho < \rho_{\rm max},\\
+ * \delta^i_j, & \rho_{\rm max} < \rho.\end{array}\right.
+ * \f}
+ *
+ * ### Inverse Jacobian
+ *
+ * This map finds the inverse Jacobian by first finding the Jacobian and then
```

Comment:
Question: given the restriction on the inverse map, imposing that restriction here would mean that the Jacobian is (I think) surprisingly simple since it basically boils down to `\xi^i = Constant(t)` for most of them, with just a `1/r^2 \partial_i r` being the only non-trivial factor. Would this be worth doing? Not necessarily now, but even later (filing a feature request)?
@markscheel have you thought about this?

Reply:
I thought about this briefly but didn't see a straightforward way to get the 1/r^2 \partial_i r term, but then again, my threshold for biting the bullet and doing some algebra is sometimes set too low. If @markscheel has an idea, I'd be happy to open an issue on this with it as a feature request.

Reply:
I looked again at this, and I think the middle case might be tricky, because it's not obvious to me you can easily solve for \xi^I before taking the derivative, because of the (rho_max/rho - 1)rho^i term

Reply:
Sorry, I'm missing something. So I'm looking at equation (19) in the dox, and these have for `r < rh_min-lamda/sqrt(4pi)` and `rho_max<r` that `\xi~r^i` which means `d\xi^i / dx^j~\delta^i_j(stuff)` where the "stuff" is different in the two cases. In the interior, you have
```
xi^i~r^i(1+A/r)/B +C^i
```
so
```
d xi^i /dx^j~\delta^i_j(1+A/r)/B+r^i(-A/r^2 dr/dx^j)/B
```
That would mean you need to figure out what `dr/dx^j` is, so
```
d r / dx^j = (d/dx^j) sqrt((x^i-C^i)\delta_{ik}(x^k-C^k))
= (1/2r) (d/dx^j) (x^i-C^i)(x^k-C^k) \delta_{ik}
= (1/2r) \delta_{ik} (\delta ^i_j(x^k -C^k) + \delta^k_j(x^i-C^i))
=  (1/2r) (\delta_{kj} (x^k -C^k) + \delta_{kj}(x^k-C^k))
=  (1/r) (x^j -C^j)
```
What I'm not sure about are the lambda factors, since `x^i` is a function of time, so there might be some weirdness there? And maybe I've made some other assumption or simplification that I didn't understand from your comments.

Reply:
You might be right, but as you said, let's leave this as an issue for a future update

Reply:
Can you please add to the documentation what we think the inverse Jacobian is and explain that it needs to be double checked? A github comment on a PR isn't really a great reminder.

I also still don't understand your statement about the rho terms. Where is the `rho^i` that I missed?

Reply:
Let me think carefully about your question about the rho terms I thought you missed and then get back to you in the issue I just opened about this: #2989

Reply:
:+1: thanks!